applications/smart-distancing/ui/utils/visualization_utils.py

```diff
@@ -390,6 +391,38 @@ def visualize_boxes_and_labels_on_image_array(
     return image
 
 
+
+def checkDistance(nn_out):
+
```

Comment:
Is it possible documenting (add a docstring to the function) if the function is finalized